qemu/tests/cfg/blockdev_commit_server_down.cfg

```diff
@@ -0,0 +1,39 @@
+- blockdev_commit_server_down:
+    only Linux
+    only filesystem
+    virt_test_type = qemu
+    type = blockdev_commit_server_down
+    qemu_force_use_drive_expression = no
+    start_vm = no
```

Comment:
Better to add `not_preprocess = yes ` to avoid preporcess again.

Reply:
done